repo/db/cafe_sessions.go

```diff
@@ -37,11 +38,12 @@ func (c *CafeSessionDB) AddOrUpdate(session *repo.CafeSession) error {
 		return err
 	}
 
+	// TODO: There are props on pb.CafeSession we're not persisting
 	_, err = stmt.Exec(
 		session.Id,
 		session.Access,
 		session.Refresh,
-		session.Expiry.UnixNano(),
+		int(session.Exp.Nanos), // Nanos or Seconds?
```

Comment:
I'm not clear about `Seconds` vs `Nanos` on a `Timestamp` object, and also know @sanderpick made some changes to get time related db things working correctly on Android. Want to highlight this and make sure we do the right think here.

Reply:
Just to be consistent w/ the other persisted data, I'd say let's use nanos here even though we don't really need that resolution. Though, I think `Exp.Nanos` is just the nano second remainder (go's `UnixNano()` returns the entire delta from epoch as nanos) because it's only `int32`, `Exp.Seconds` is `int64`. We want `int64` nanos here. Maybe easiest to convert to a go `Time` then use `UnixNano()`.